src/routes/search/searchHelpers.ts

```diff
@@ -35,7 +37,11 @@ export const searchHitsTemplate: TemplateWithBindEvent<Hit<{ url?: string }>> =
             </a>
           </h2>
           <div class="post-excerpt">
-            ${getHighlightOrSnippetResultValue(hit._highlightResult?.metaDescription)}
+            ${getHighlightOrSnippetResultValue(
+              hit?.entryType === "Service Entry"
+                ? hit._snippetResult?.metaDescription
+                : hit._highlightResult?.metaDescription,
+            )}
```

Comment:
this is getting utilized in tandem with [these settings in Algolia](https://dashboard.algolia.com/apps/9IIDYROXZ5/explorer/configuration/prod_ldaf/snippeting). Since some of the service entry plain text descriptions can get pretty length, I'm limiting snippets in any search hits to 50 words. Unfortunately you can't limit `highlightResult`, so adding in a check to only do this for "Service Entry" content